clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/OperationsController.groovy

```diff
@@ -51,6 +53,7 @@ class OperationsController {
   @Autowired OrchestrationProcessor orchestrationProcessor
   @Autowired Registry registry
   @Autowired (required = false) Collection<AllowedAccountsValidator> allowedAccountValidators = []
+  @Autowired (required = false) Collection<AtomicOperationDescriptionPreProcessor> atomicOperationDescriptionPreProcessors = []
```

Comment:
would be worth making this a `List` - Spring will respect an `@Order` annotation on any preprocessor and the List would end up sorted by the `@Order` items. If for some reason we want a lower or higher precedence pre-processor we would get that for free